pkg/schema/schema.json

```diff
@@ -75,6 +80,12 @@
                             }
                         ]
                     }
+                },
+                "topLevelDomains": {
+                    "type": "array",
```

Comment:
you can supply:

```
     "type": "array",
      "items": {
          "type": "string"
      },
```

that seems to work with the `K8SOmission.Namespaces`, so should be supported by the generator.

Reply:
But then the generated schema is not correct because the default values are of `[]interface{}` type.

Reply:
the enums you mean? what a shame, I guess that's another patch for the other library for me then

Reply:
https://github.com/atombender/go-jsonschema/pull/31

I guess we have to vendor my fork until we can have it in the "upstream" repo. I send a separate PR.